client/src/app/core/ui-services/login-data.service.ts

```diff
@@ -155,32 +155,33 @@ export class LoginDataService {
      */
     private async setup(): Promise<void> {
```

Comment:
I think this service is not needed anymore. It was introduced to OS3 because on the login mask (or for every not-logged in user and no guests allowed), you had no access to the autoupdate, so you could not get the data needed for the login page. We solved this by adding a get-route in the server returning only the login data needed. This services unified the two sources of login data: One source is the get request, but when you are logged in the autoupdate is the second source of data. Also the data was stored in the local storage, so the following startups do not take so long (this is actually a third source used for initializing this service;) )

Now, the autoupdate for the orga and resource is always open, also for not authenticated users. Especially all "login data" is now available through the autoupdate. Also, for the first try, we should not cache this data to make the lifes in the client easier.

I propose just deleting this service and directly use the orgasettingsservice where these pieces of data are needed.

Reply:
Okay